src/elf.rs

```diff
@@ -25,24 +25,24 @@ pub enum Relro {
 impl fmt::Display for Relro {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match *self {
-            Relro::None => write!(f, "None"),
+            Relro::None => write!(f, "None   "),
```

Comment:
Would prefer to use formatting parameters for alignment instead static string. eg. `"{:7}"`  or `"{:<7}"`